ragna/deploy/_ui/central_view.py

```diff
@@ -336,9 +336,16 @@ def on_click_source_info_wrapper(self, event, sources):
         ]
         for rank, source in enumerate(sources, 1):
             location = source["location"]
+            pages = source["document"].get("pages")
```

Comment:
`source["document"]` does not have a field called `pages`. You can find the schemas here

https://github.com/Quansight/ragna/blob/38f75f6d88e74f72c030ef50045315b81eb203aa/ragna/deploy/_api/schemas.py#L35-L39

https://github.com/Quansight/ragna/blob/38f75f6d88e74f72c030ef50045315b81eb203aa/ragna/deploy/_api/schemas.py#L18-L20

The `location` field is what you want. You can check if it is a non-empty string and if yes, prepend `page(s)` to it. You can use the following existing block for it:

https://github.com/Quansight/ragna/blob/38f75f6d88e74f72c030ef50045315b81eb203aa/ragna/deploy/_ui/central_view.py#L339-L340